src/main/kotlin/platform/forge/creator/ForgeProjectSettingsWizard.kt

```diff
@@ -78,6 +80,13 @@ class ForgeProjectSettingsWizard(private val creator: MinecraftProjectCreator) :
         mcpWarning.isVisible = (mcpVersionBox.selectedItem as? McpVersionEntry)?.isRed == true
     }
 
+    private val forgeVersionBoxListener = ActionListener {
+        val supportedMixinVersion = forgeVersionBox.selectedItem as SemanticVersion >= SemanticVersion.parse("31.2.45")
+        mixinsCheckbox.isEnabled = supportedMixinVersion
+        mixinsCheckbox.isSelected = supportedMixinVersion
+
```

Comment:
The first line is too long, and might fail if `forgeVersionBox.selectedItem` for some reason.
`SemanticVersion.release` can also be used for hardcoded versions:
```suggestion
        val selectedVersion = forgeVersionBox.selectedItem as? SemanticVersion ?: return@ActionListener
        val supportedMixinVersion = selectedVersion >= SemanticVersion.release(31, 2, 45)
        mixinsCheckbox.isEnabled = supportedMixinVersion
        mixinsCheckbox.isSelected = supportedMixinVersion
```

Other than that it looks good enough, the "simple" comparison used here does the job for me, I don't expect people to use forge versions that are more than 6 months old.